visualizations/fan_chart/tests/test_fan_chart.py

```diff
@@ -20,30 +22,66 @@
     'error': [2]
 }
 
+working_directory = os.path.dirname(os.path.abspath(__file__))
+
 
 class TestFanChart(unittest.TestCase):
+    def test_using_csv(self):
+        file = os.path.join(working_directory, 'obs_test.csv')
```

Comment:
Instead of adding separate csv files to test with, you should probably send the the csv data as StringIO objects as that can more easily be generated. See for instance `visualization/plotly/tests/test_filtered_plotly.py`.